EIPS/eip-4494.md

```diff
@@ -0,0 +1,207 @@
+---
+eip: 4494
+title: Permit for ERC-721 NFTs
+description: ERC-712-singed approvals for ERC-721 NFTs
+author: Simon Fremaux (@dievardump), William Schwab (@wschwab)
+discussions-to: https://ethereum-magicians.org/t/eip-extending-erc2612-style-permits-to-erc721-nfts/7519/2
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-11-25
+requires: 165, 712, 721
+---
+
+## Abstract
+The "Permit" approval flow outlined in [ERC-2612](./eip-2612.md) has proven a very valuable advancement in UX by creating gasless approvals for ERC20 tokens. This EIP extends the pattern to ERC-721 NFTs. This EIP borrows heavily from ERC-2612.
```

Comment:
```suggestion
The "Permit" approval flow outlined in [ERC-2612](./eip-2612.md) has proven a very valuable advancement in UX by creating gasless approvals for ERC-20 tokens. This EIP extends the pattern to ERC-721 NFTs. This EIP borrows heavily from ERC-2612.
```

Reply:
ooh, I missed one, I'll get it in the next push